src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+@use './variables.scss' as *;
+
+.calendar {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1px;
+    width: calc(#{$calendar-day--size * 7} + 26px); //  26px == 20px padding + 6px gaps
+    padding: 10px;
+
+    @each $day in $days {
+      &--start-day-#{$day} .calendar__day:first-child {
+        margin-left: #{$sum}px;
+      }
+
+      $sum: calc($sum + $gap);
+    }
```

Comment:
The variable `$sum` is used in the loop but is not initialized before the loop starts. This could lead to unexpected results. Consider initializing `$sum` before the loop.